packages/eslint-plugin/docs/rules/prefer-string-starts-ends-with.md

```diff
@@ -0,0 +1,54 @@
+# Enforce to use `String#startsWith` and `String#endsWith` over other options (prefer-string-starts-ends-with)
```

Comment:
Non-blocking suggestion:

```suggestion
# Enforce the use of `String#startsWith` and `String#endsWith` instead of other equivalent methods of checking substrings (prefer-string-starts-ends-with)
```